src/f32-raddstoreexpminusmax/rvv-rr2-p6.c.in

```diff
@@ -0,0 +1,140 @@
+// Copyright 2023 SiFive, Inc.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+$assert LMUL >= 1
+$LMUL_HALF = {1: "f2", 2: "1", 4: "2", 8: "4"}[LMUL]
```

Comment:
Consider dropping f2 support... performance will always be better on m1 or higher

Reply:
OK, I'll drop 1 & 8 cases.
1:  use "f2".
8: introduce vector register spill => we should avoid this.

Reply:
Solved in commit https://github.com/google/XNNPACK/pull/5876/commits/e72f6da9b7af8df1a9e97f076d2072ff7d1ac4ef